docs/01-module-block-1/02-learning-goals.adoc

```diff
@@ -16,8 +16,7 @@ wie wichtig einzelne Aspekte des Lernziels sind. Es kann hier bereits auf Litera
 
 // tag::EN[]
 [[LG-1-1]]
-==== LG 1-1: Propositional Logic
-Know the basic structure of propositional logic:
+==== LG 1-1: Know the basic structure of propositional logic
```

Comment:
Shouldn't we then also call this "Basic structure of propositional logic" here, if we're not supposed to use the operators?

Reply:
It's not forbidden to use those verbs. 🙂 
The idea is as follows: someone participates in this training and they learn all the learning goals. So, after this training, they:
- Know the basic structure of XY (this learning goal)
- Understand the limitations of a pure technical approach to software (I made this up)

Or they are able to
- Facilitate meetings with the Seismian-Falter-Method (I also made this up)


And so on. Does this make sense? 
The heading of the learning goal should already make it obvious what people learn, the text within can/should be used to add more details.

Reply:
Understood.

Reply:
Taking this as "resolved". Feel free to re-open, @larsrh or @programming-wolf .